hugr-core/src/hugr/linking.rs

```diff
@@ -0,0 +1,520 @@
+//! Directives and errors relating to linking Hugrs.
+
+use std::{collections::HashMap, fmt::Display};
+
+use itertools::Either;
+
+use crate::{
+    Hugr, HugrView, Node,
+    core::HugrNode,
+    hugr::{HugrMut, internal::HugrMutInternals},
+};
+
+/// Methods for linking Hugrs, i.e. merging the Hugrs and adding edges between old and inserted nodes.
```

Comment:
A bit shorter so it fits in 80chars

```suggestion
/// Methods that merge Hugrs, adding edges between old and inserted nodes.
```